### PR TITLE
server: Add a /info endpoint

### PR DESCRIPTION
**Pull request by mrunalp**
~~This depends on #816.~~

### CODE REVIEW

**Comment by mrunalp** [comment]
Rebased on master as #816 is merged.

**Comment by runcom** [comment]
Looks ok, I'll figure out what's happening in #814 and then pull everything into a working branch to start the cadvisor stuff

File: server/inspect.go [hunk 1]
```diff
@@ -17,11 +17,31 @@ type ContainerInfo struct {
...
 }
 
-// StartInspectEndpoint starts a http server that
-// serves container information requests
-func (s *Server) StartInspectEndpoint() error {
+// CrioInfo stores information about the crio daemon
+type CrioInfo struct {
+	StorageDriver string `json:"storage_driver"`
+	StorageRoot   string `json:"storage_root"`
+}
```
Comment (runcom) [inline server/inspect.go]: Need a version info here (needed by cadvisor handler)

Comment (runcom) [reply server/inspect.go]: probably in a follow up

**Pull request merged**